docs/.gitignore

```diff
@@ -1 +1,12 @@
-sql-configs.html
+generated-agg-funcs-examples.html
+generated-agg-funcs-table.html
+generated-array-funcs-examples.html
+generated-array-funcs-table.html
+generated-datetime-funcs-examples.html
+generated-datetime-funcs-table.html
+generated-json-funcs-examples.html
+generated-json-funcs-table.html
+generated-map-funcs-examples.html
+generated-map-funcs-table.html
+generated-sql-configuration-table.html
+generated-window-funcs-table.html
```

Comment:
Every time we add a function group, adding a new entry here is a bit annoying , so how about having the same prefix in common and putting a single entry here like `generated-builtin-*.html`?